h/static/scripts/annotator/guest.coffee

```diff
@@ -68,7 +70,6 @@ module.exports = class Guest extends Annotator
             this.publish(event, args)
       formatter: (annotation) =>
         formatted = {}
-        formatted.uri = @getHref()
```

Comment:
This is orthogonal to these changes. I can restore this if you all want. On the backend, we're indexing this as an alias for `target.source` anyway.